assets/src/components/v2/dup/dup_free_text.tsx

```diff
@@ -3,7 +3,8 @@ import _ from "lodash";
 
 import { classWithModifier, classWithModifiers, imagePath } from "Util/util";
 
-const pillIcons = ["red", "blue", "orange", "green", "silver"];
+const textPills = ["red", "blue", "orange", "green", "silver"];
+const iconPills = ["cr", "bus"];
```

Comment:
Ohhhhhh phoo... I just noticed that this file is an almost-duplicate of another v2 `free_text.tsx` file. Would you be interested in consolidating? If not, can make a clean-up task. I shouldn't have introduced this `dup_free_text.tsx` file. My bad

Reply:
I don't mind doing it. I think everything v2 is using the same one now. I tested some of the other apps (`ReconstructedTakeover` uses `FreeText`) and it looks good.

Reply:
Correct. We do technically get alerts for predictions, but it would only render if `alert_is_inline?/1` is `true` and that is hardcoded to `false` for now. I think it was left in and hardcoded to `false` so bus shelter alerts could be revisited, but that didn't happen as soon as we thought.